bluenrg_family/bluenrg_3/CMakeLists.txt

```diff
@@ -24,6 +24,8 @@ endif()
 
 zephyr_compile_definitions(
   -DUSE_FULL_LL_DRIVER
+  -DCONFIG_HW_LS_XTAL
+  -DCONFIG_HW_SMPS_10uH
```

Comment:
Should these be part of DTS ?

Reply:
It is possible to handle them there, especially the first one; however, we can postpone it to the future.